api.planx.uk/modules/pay/helpers.ts

```diff
@@ -36,15 +39,23 @@ export async function logPaymentStatus({
       context: { sessionId, flowId, teamSlug },
     });
   } else {
-    // log payment status response
     try {
+      // get fee breakdown for this session
+      const sessionData = await getSessionData(sessionId);
+      const passportData = sessionData?.passport?.data;
+
+      let feeBreakdown: FeeBreakdown | undefined;
+      if (passportData) feeBreakdown = getFeeBreakdown(passportData);
```

Comment:
nit: There's no reason we should get to this stage without `passportData` - maybe a better way to type guard this would be to throw an error?

Reply:
Yep good shout - updated :heavy_check_mark: